package.json

```diff
@@ -45,13 +50,19 @@
     "prettier": "^2.2.1",
     "rxjs": "^6.4.0",
     "serve": "^14.2.3",
+    "chokidar-cli": "^3.0.0",
+    "live-server": "^1.1.0",
+    "sass": "^1.51.0",
+    "parallelshell": "3.0.1",
+    "shelljs": "^0.8.5",
     "typescript": "^4.1.3"
   },
   "dependencies": {
     "axios": "^0.18.0",
     "jest": "^26.6.3",
     "ramda": "^0.26.1",
     "rollup": "^4.18.0",
-    "ts-jest": "^26.4.4"
+    "ts-jest": "^26.4.4",
+    "sfeir-school-theme": "^3.1.0"
```

Comment:
ça apporte quoi de tout fusionner ? c'était bien de tout séparer je trouve et au pire avoir un package.json à la racine qui sert de workspace

Reply:
great idea, let me refactor buddy

Reply:
IMHO: no need to move everything into an apps folder. This move only create noise and does make the repo less clear for students... You can do the same thing without moving steps and docs directories into an app folder and keep this repo uniform with the template and other schools